core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt

```diff
@@ -270,63 +273,71 @@ fun LPGenericElevatedCard(
  */
 @Composable
 fun LPGenericElevatedCardImage(
```

Comment:
If you are going to modify a function that it is already created, please add your name in the function documentation like this: `@modifyBy YourName`